style.css

```diff
@@ -91,3 +104,185 @@ button {
 }
 
 /* ==End of Introduction section== */
+
+/* ==Beginning of Projects section== */
+
+#projects > h2 {
+  margin-left: 16px;
+  margin-top: 40px;
+  margin-bottom: 32px;
+}
+
+#projects .project-name {
+  background-color: #1a2236;
+  width: 100%;
+  height: 156px;
+  margin-top: -6px;
+}
+
+#projects .project-division > img {
+  width: 100%;
+  height: auto;
+}
+
+#projects h4 {
+  font-family: 'Poppins', sans-serif;
+  font-size: 20px;
+  line-height: 80%;
+  font-weight: 300;
+  font-style: normal;
+  color: #fff;
+  padding: 20px 36px 16px 20px;
+}
+
+#projects ul {
+  display: flex;
```

Comment:
### OPTIONAL
- [ ] For DRY CSS you should combine duplicate properties. Example, you can create class `.d-flex { display: flex}` and use this class in HTML file.

- [More information about DRY principles](https://www.freecodecamp.org/news/7-important-tips-for-writing-better-css/) 

- [Html and css best practices in curriculum](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)